docs/samples/custom-domain/README.md

```diff
@@ -0,0 +1,128 @@
+# Setting up custom domain name
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. You have a custom domain configured to route incoming traffic either to the Cloud provided Kubernetes Ingress gateway or the istio-ingressgateway / kfserving-ingressgateway's IP address / Load Balancer.
+
+## Create the Ingress resource
+
+#### Note: This step is only necessary if using a domain that is configured to route incoming traffic to the cluster's Kubernetes Ingress. For example, many cloud platforms provide default domains which route to a Kuberenetes Ingress. If using a domain that is routed to the `istio-ingressgateway`, you can skip this step.
+
+#### Note: Use `kfserving-ingressgatway` instead of `istio-ingressgateway` as your `INGRESS_GATEWAY` if you are deploying KFServing as part of Kubeflow install, and not independently.
+
+Edit the `kfserving-ingress.yaml` file to add your custom wildcard domain to the `spec.rules.host` section, replacing `<*.custom_domain>` with your custom wildcard domain. This is so that all incoming network traffic from your custom domain and any subdomain is routed to the `istio-ingressgateway`.
+
+```
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: kfserving-ingress
+  namespace: istio-system
+spec:
+  rules:
+    - host: "<*.custom_domain>"
+      http:
+        paths:
+          - backend:
+              serviceName: istio-ingressgateway
+              servicePort: 80
+            path: /
+```
+
+Apply the Ingress resource
+
+```
+kubectl apply -f kfserving-ingress.yaml
+```
+
+Expected Output
+
+```
+$ ingress.networking.k8s.io/knative-ingress created
+```
+
+## Modify the config-domain Configmap
+
+Modify the config map to use your custom domain when assigning hostnames to Knative services.
+
+Open the `config-domain` configmap to start editing it.
+
+```
+ kubectl edit configmap config-domain -n knative-serving
+```
+
+Specify your custom domain in the `data` section of your configmap and remove the default domain that is set for your cluster.
+
+```
+apiVersion: v1
+kind: ConfigMap
+data:
+  <custom_domain>: ""
+metadata:
+...
+```
+
+Save your changes. Expected Output
+
+```
+configmap/config-domain edited
+```
+
+With your Ingress routing rules and Knative configmaps set up, you can create Knative services that use your custom domain.
+
+## Create a sample InferenceService
+
+To create an InferenceService using Tensorflow, refer to the [guide](https://github.com/kubeflow/kfserving/tree/master/docs/samples/tensorflow).
```

Comment:
Maybe we should use a relative path for this link?

Reply:
Used the relative path for the tensorflow example